crates/holochain/src/sweettest/sweet_conductor_batch.rs

```diff
@@ -178,6 +218,17 @@ impl SweetConductorBatch {
             c.force_all_publish_dht_ops().await;
         }
     }
+
+    /// Wait for the first conductor to have started gossipping. If all conductors in this batch were
+    /// started at the same time, this should be enough to ensure that all conductors have started gossipping.
+    /// If other conductors are added later, separately check that they have started gossipping as required.
+    pub async fn require_initial_gossip_activity_for_cell(
+        &self,
+        cell: &SweetCell,
+        timeout: Duration,
+    ) {
+        SweetConductor::require_initial_gossip_activity_for_cell(&self.0[0], cell, timeout).await;
```

Comment:
If not for the change added in #2183 (which I questioned in a comment above), it would be easy to wait for all conductors in parallel, but as it stands it would require including a list of cells from each conductor, which is pretty annoying. It seems that it would be good to wait for all of them, otherwise we're just decreasing our chances of not running into this problem, but not fully eliminating the chance.